ast Module has no attribute "JoinedStr"

Could it be that there is a gap for some of the newer python features?

```bash
$ cat joinedstr_test.py
import ast

def walker(tree: ast.Module):
    for node in ast.walk(tree):
        if isinstance(node, ast.FunctionDef):
            print("function", node)
        if isinstance(node, ast.JoinedStr):
            print("joinedstr", node)
$ mypy joinedstr_test.py
joinedstr_test.py:7: error: Module has no attribute "JoinedStr"
$ python --version
Python 3.6.5 :: Anaconda, Inc.
$ pip freeze | grep mypy
mypy==0.600
$ python -c "import ast;ast.JoinedStr"
$ echo $?
0
```

--- a/stdlib/3/_ast.pyi
+++ b/stdlib/3/_ast.pyi
@@ -233,6 +233,15 @@ class Num(expr):
 class Str(expr):
     s = ...  # type: str
 
+if sys.version_info >= (3, 6):
+    class FormattedValue(expr):
+        value = ...  # type: expr
+        conversion = ...  # type: typing.Optional[int]
+        format_spec = ... # type. typing.Optional[expr]
+        
+    class JoinedStr(expr):
+        values = ...  # type: typing.List[expr]
+
 class Bytes(expr):
     s = ...  # type: bytes
 

--- a/stdlib/3/_ast.pyi
+++ b/stdlib/3/_ast.pyi
@@ -237,8 +237,8 @@ if sys.version_info >= (3, 6):
     class FormattedValue(expr):
         value = ...  # type: expr
         conversion = ...  # type: typing.Optional[int]
-        format_spec = ... # type. typing.Optional[expr]
-        
+        format_spec = ...  # type. typing.Optional[expr]
+
     class JoinedStr(expr):
         values = ...  # type: typing.List[expr]
 

